src/test/kotlin/no/nav/syfo/oppfolgingstilfelle/domain/PersonOppfolgingstilfelleSpek.kt

```diff
@@ -0,0 +1,100 @@
+package no.nav.syfo.oppfolgingstilfelle.domain
+
+import io.ktor.util.*
+import no.nav.syfo.clients.syketilfelle.domain.KOppfolgingstilfellePerson
+import no.nav.syfo.clients.syketilfelle.domain.KSyketilfelledag
+import no.nav.syfo.domain.AktorId
+import no.nav.syfo.domain.Fodselsnummer
+import org.amshove.kluent.shouldBeEqualTo
+import org.spekframework.spek2.Spek
+import org.spekframework.spek2.style.specification.describe
+import testutil.UserConstants.ARBEIDSTAKER_FNR
+import testutil.generator.generateKOppfolgingstilfelle
+import java.time.LocalDate
+
+@InternalAPI
+object PersonOppfolgingstilfelleSpek : Spek({
+
+    describe("isCandidateForPrediction") {
```

Comment:
Kan du ha litt mer beskrivende `it`er? Nå står det det samme i alle 🤷‍♂️

Reply:
Jepp! Glemte å endre det 👍

Reply:
Har endret beskrivelser for tester nå.